### ISSUE TITLE
Compilation warning "Struct 'ETEXT_DESC' was previously declared as a class"

### ISSUE THREAD

**Issue by y-fedorov** [tesseract-ocr/tesseract#414]
` ..../include/tesseract/baseapi.h:594:21: Struct 'ETEXT_DESC' was previously declared as a class`

In baseapi.h declared as fallows:
`class ETEXT_DESC;`
...
```
/**
   * Make a HTML-formatted string with hOCR markup from the internal
   * data structures.
   * page_number is 0-based but will appear in the output as 1-based.
   * monitor can be used to
   * 	cancel the recognition
   * 	receive progress callbacks
   */
  char* GetHOCRText(struct ETEXT_DESC* monitor, int page_number);
```

Do we really need struct definition here?

My system:
Xcode 7.3.1
Build version 7D1014

Clang:
Apple LLVM version 7.3.0 (clang-703.0.31)
Target: x86_64-apple-darwin15.6.0
Thread model: posix

**Comment by stweil** [comment]
I just sent pull request #415 which fixes this issue.

**Comment by zdenop** [comment]
fixed by #415
